src/components/CodeHighlight/CodeHighlight.js

```diff
@@ -0,0 +1,61 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+
+import * as Languages from './Languages';
+import * as Themes from './Themes';
```

Comment:
```suggestion
import {
  THEMES,
  themes,
} from './Themes';
```
Please change import and remove destructuring below

Reply:
Done

Reply:
Done